### PR TITLE
Homework2: Mikalai Hutsalenka

### PR DESCRIPTION
**Pull request by vorongn**

### CODE REVIEW

File: src/dop_task04_fibonachi.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# ряд фибаначи
+#  end=' ' в строчку пусть выводит
+# вопрос: начинать с 0 как в wiki или с 1 как в школьном учебнике?
```
Comment (belovmd) [inline src/dop_task04_fibonachi.py:3]: as you want

File: src/task01_cena_kol_vo.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+m = int(input('введите стоимость рублей:'))
+n = int(input('введите стоимость копеек:'))
+ll = int(input('введите к-во товара:'))
```
Comment (belovmd) [inline src/task01_cena_kol_vo.py]: why ll?

File: src/task02_02_function.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# 1. on CheckiO your solution should be a function
+# 2. the function should return the right answer, not print it.
```
Comment (belovmd) [inline src/task02_02_function.py]: please put task description

File: src/task02_05_peremnojit_vse_krome_nul.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
+def checkio(number: int) -> int:
```
Comment (belovmd) [inline src/task02_05_peremnojit_vse_krome_nul.py:18]: please put task description it's unclear what to check

File: src/task03_samoe_dlinnoe.py [hunk 5]
```diff
@@ -0,0 +1,13 @@
+my_string = '1 22 333 4444 55555 55555, 333. 666666.....  333'
```
Comment (belovmd) [inline src/task03_samoe_dlinnoe.py]: unclear task. please put task description

File: src/task04_dell_udalit_probeli_povtori.py [hunk 6]
```diff
@@ -0,0 +1,13 @@
+stroka = str(input('введите строку:'))
+
+
+# stroka1 = stroka.replace(' ', '') - можно заменить на разделить/склеить
```
Comment (belovmd) [inline src/task04_dell_udalit_probeli_povtori.py]: why do we need this comment

File: src/task04_dell_udalit_probeli_povtori.py [hunk 7]
```diff
@@ -0,0 +1,13 @@
+stroka = str(input('введите строку:'))
```
Comment (belovmd) [inline src/task04_dell_udalit_probeli_povtori.py]: please put task descriptions

File: src/task05_kol_strochnih_propinih.py [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+
+# определили шаблон
+stroka2 = alfavit.sub('', stroka)
+
+
+# отфильтровали по ркгулярному выражению
+for i in stroka2:
+    if i.isupper():
+        propisnie += 1
+    else:
```
Comment (belovmd) [inline src/task05_kol_strochnih_propinih.py]: elif?

**Review by belovmd — COMMENTED** [review]
1. please put tasks description to docstring or . comments for all tasks. 
2. please address these issues

File: src/task05_kol_strochnih_propinih.py (hunk 8 above)
Comment (vorongn) [reply src/task05_kol_strochnih_propinih.py]: remade on elif

**Comment by vorongn** [comment]
Added a description to the tasks.

File: src/task01_cena_kol_vo.py (hunk 2 above)
Comment (vorongn) [reply src/task01_cena_kol_vo.py]: completely redone

File: src/dop_task02_prime_number.py [hunk 9]
```diff
@@ -0,0 +1,11 @@
+# простое число
+num = int(input('Введите число: '))
+if num > 1:
+    for i in range(2, num - 1):
```
Comment (belovmd) [inline src/dop_task02_prime_number.py:4]: sqrt

File: src/task03_samoe_dlinnoe.py [hunk 10]
```diff
@@ -0,0 +1,19 @@
+"""
+Найти самое длинное слово в введенном предложении.
+Учтите что в предложении есть знаки препинания.
+"""
+
+
+my_string = '1 22 333 4444 55555 55555, 333.! 666666.....  333???????'
+prep = ',.?!'
```
Comment (belovmd) [inline src/task03_samoe_dlinnoe.py:8]: ()[]{};:

**Review by belovmd — COMMENTED** [review]
9

**Pull request closed without merging**